include/zephyr/bluetooth/mesh/blob_cli.h

```diff
@@ -419,6 +420,13 @@ uint8_t bt_mesh_blob_cli_xfer_progress_active_get(struct bt_mesh_blob_cli *cli);
  */
 bool bt_mesh_blob_cli_is_busy(struct bt_mesh_blob_cli *cli);
 
+/** @brief Set chunk sending interval in ms
```

Comment:
Description is not sufficient. It should include which blob model mode is used, restrictions. Couple of words about reasoning of this non-standard API function. It is also required to mention this is an optional API. Right now it is not clear at all.